webapp/src/ts/constants.ts

```diff
@@ -6,6 +6,25 @@
 // go and if it's still too long then API will respond with a 413.
 const BODY_LENGTH_LIMIT = 32000000; // 32 million
 
+// To keep long _changes requests open, API sends a heartbeat (a new line) every 10 seconds, starting on the 10sh
+// second. Sending data through automatically adds a success 200 status, even though the results of the request is
+// not known, and it can still fail.
+// PouchDb only checks the request status to determine whether the request is successful or not. \
+// In order to enable a retry, and not incorrectly communicate success to the client, validate _changes request
+// responses and update the respons object accordingly before passing it to PouchDb for processing.
```

Comment:
I noticed the typos in this comment in a re-read. To optimize on CI time, I'm going to push the fixes to typos along with the review feedback.